src/libraries/pkg/baseline/packageIndex.json

```diff
@@ -549,7 +549,8 @@
       "BaselineVersion": "5.0.0",
       "InboxOn": {},
       "AssemblyVersionInPackageVersion": {
-        "3.1.4.0": "5.0.0",
+        "3.1.3.0": "3.1.3",
+        "3.1.4.0": "3.1.4",
```

Comment:
dont we increment the last part in the servicing releases ?

Reply:
Maybe but this is just reacting to a package serviced in release/3.1 which was not yet part of libraries so may use different versioning

Reply:
I’m not sure I understand the question.

This library was not part of corefx. This is the first release it is being built using the “corefx/libraries” packaging infrastructure.

Reply:
He is saying that usually in servicing we only increment  the last part of the assembly version, so package 3.1.4 should have had assembly version 3.1.3.1 instead of 3.1.4.0, but that wasn’t a decision made on this PR, this is just reacting to what already happened in servicing

Reply:
yes i should have been more clearer, i meant to ask if the versioning scheme used here is on purpose or just a mistake in packaging (nothing wrong with this pr)

Reply:
@Anipik keep in mind that at least in release/3.1 this package is being built on a different repo (coresetup) which means that they were using a slightly different infrastructure and were possibly not following the same versioning guidelines we do in corefx. Now we have moved it to runtime it should start following conventions starting from 5.0 release